src/component/BpmnVisualization.ts

```diff
@@ -92,4 +317,171 @@ export class BpmnVisualization {
+    this.graph.getView().addListener(mxgraph.mxEvent.UNDO, listener);
+  }
+
+  updateStyle<C extends string>(bpmnElementIds: string | string[], style: ShapeStyleUpdate<C> | EdgeStyleUpdate<C>): void {
+    this.graph.getModel().beginUpdate();
+    try {
+      ensureIsArray<string>(bpmnElementIds).forEach(bpmnElementId => {
+        const cell = this.graph.getModel().getCell(bpmnElementId);
+        let cellStyle = cell.getStyle();
+
+        console.log('old style = %s', cellStyle);
+        console.log(`---- new style -----`);
+        console.log(style);
+
+        cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_OPACITY, style.opacity);
+        cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_SHADOW, Number(style.withShadow));
+
+        const font = style.font;
+        if (font) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FONTCOLOR, font.color);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FONTSIZE, font.size);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FONTFAMILY, font.family);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_FONTSTYLE, BpmnVisualization.getFontStyleValue(font));
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_TEXT_OPACITY, font.opacity);
+        }
+
+        const label = style.label;
+        if (label) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_WIDTH, label.width);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_POSITION, label.horizontalPosition);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_VERTICAL_LABEL_POSITION, label.verticalPosition);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_BACKGROUNDCOLOR, label.backgroundColor);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_BORDERCOLOR, label.borderColor);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_LABEL_PADDING, label.padding);
+        }
+
+        const stroke = style.stroke;
+        if (stroke) {
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKECOLOR, stroke.color);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKEWIDTH, stroke.width);
+          cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_STROKE_OPACITY, stroke.opacity);
+
+          if (stroke.dashPattern) {
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_DASHED, 1);
+            cellStyle = mxgraph.mxUtils.setStyle(cellStyle, mxgraph.mxConstants.STYLE_DASH_PATTERN, stroke.dashPattern);
+          }
```

Comment:
**chore:** 
We choose to not implement it in the first step.
According the need, we can add it in a next step.